test/fixtures/generate/fixtures.geojson

```diff
@@ -0,0 +1,696 @@
+{
```

Comment:
The osm file has all the fixtures in one file. I left the geojson since it's easier to use that to split features into separate files. Happy to save it in individual geojsons if that would be better.